components/classes/travel-preferences.schema.json

```diff
@@ -0,0 +1,208 @@
+{
+  "meta:license": [
+    "Copyright 2020 Adobe Systems Incorporated. All rights reserved.",
+    "This work is licensed under a Creative Commons Attribution 4.0 International (CC BY 4.0) license",
+    "you may not use this file except in compliance with the License. You may obtain a copy",
+    "of the License at https://creativecommons.org/licenses/by/4.0/"
+  ],
+  "$id": "https://ns.adobe.com/xdm/classes/travel-preferences",
```

Comment:
I think we actually want this to be a mixin, extending profile class. 

So `Travel Preference Details` Mixin- extending profile Class.

Reply:
Take a look at the updates I just pushed and see if this is what you expected.